### PR TITLE
Fix Github Actions of Tauri Plugin with-api template

### PR DESCRIPTION
**Pull request by Miniontoby**
### What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [X] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [X] I have added a convincing reason for adding this feature, if necessary

### Other information

check https://discord.com/channels/616186924390023171/1090989894991618171 for more info I guess

### CODE REVIEW

**Review by lucasfernog — DISMISSED** [review]

**Comment by lucasfernog** [comment]
Can you add a change file like this one? https://raw.githubusercontent.com/tauri-apps/tauri/dev/.changes/cli-sign-non-interactive.md

**Comment by FabianLars** [comment]
Also might as well update actions/checkout to v3 while we're at it.

**Comment by Miniontoby** [comment]
> Also might as well update actions/checkout to v3 while we're at it.

I will do that as well tomorrow for all templates

**Comment by Miniontoby** [comment]
> Can you add a change file like this one? https://raw.githubusercontent.com/tauri-apps/tauri/dev/.changes/cli-sign-non-interactive.md

Never used that type of stuff before.
Btw it said "A change file is added if any packages will require a version bump due to this PR", but there are no packages to be bumped!

**Comment by FabianLars** [comment]
The cli will be bumped for this (patch). Here's an example change file for cli changes: https://github.com/tauri-apps/tauri/blob/dev/.changes/cli-info.md

**Comment by Miniontoby** [comment]
Oke, I will look at it tomorrow and then I will also have my commits signed.

**Comment by Miniontoby** [comment]
> The cli will be bumped for this (patch). Here's an example change file for cli changes: https://github.com/tauri-apps/tauri/blob/dev/.changes/cli-info.md

What files should I mention in it and how?

**Comment by Miniontoby** [comment]
Let me know if anything else should be changed as well or else just merge it if possible

**Review by FabianLars — DISMISSED** [review]

**Review by FabianLars — DISMISSED** [review]

**Comment by Miniontoby** [comment]
Maybe I should add a permissions block to the audit workflow(s) so the users don't have to manually change stuff for the audit token to be able to do what it is supposed to do.

File: .changes/cli-templates.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"cli.rs": "patch"
```
Comment (lucasfernog) [inline .changes/cli-templates.md:2]:
```suggestion
"cli.rs": "patch"
"cli.js": "patch"
```

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**